drv/sidecar-mainboard-controller/src/tofino2.rs

```diff
@@ -231,3 +232,435 @@ impl Sequencer {
         })
     }
 }
+
+bitfield! {
+    #[derive(Copy, Clone, PartialEq, FromPrimitive, AsBytes)]
+    #[repr(C)]
+    pub struct DebugPortState(u8);
+    pub send_buffer_empty, set_send_buffer_empty: 0;
```

Comment:
Do we need any of these setters (`set_send_buffer_empty`, `set_receive_buffer_empty`, etc)?

Reply:
I used these while debugging the FPGA logic. They're currently not used, but we may have some clean-up logic in the future which may want to use these.